haproxy/tox.ini

```diff
@@ -24,14 +24,12 @@ deps =
 setenv =
   HAPROXY_LEGACY=false
   legacy: HAPROXY_LEGACY=true
-  16: HAPROXY_VERSION=1.6.15
-  17: HAPROXY_VERSION=1.7.12
-  18: HAPROXY_VERSION=1.8.27
-  20: HAPROXY_VERSION=2.0.20
-  21: HAPROXY_VERSION=2.1.11
-  22: HAPROXY_VERSION=2.2.9
-  23: HAPROXY_VERSION=2.3.5
-  24: HAPROXY_VERSION=2.4-dev10
+  17: HAPROXY_VERSION=1.7.13
+  18: HAPROXY_VERSION=1.8.29
+  20: HAPROXY_VERSION=2.0.21
+  22: HAPROXY_VERSION=2.2.11
+  23: HAPROXY_VERSION=2.3.8
+  24: HAPROXY_VERSION=2.4-dev13
```

Comment:
Should we use `latest` instead?

Reply:
unsure, as sometimes, there are some changes which can break tests. WDYT?

Reply:
Usually we don't test against `latest`, but it might be worth to have a team discussion about it (not just about haproxy, but other integration as well), as it could prevent some support cases.

I'm still merging this pr.

Reply:
understood. I'm not against it as well.